### PR TITLE
Feat rspy99/jupyterhub

### PR DESCRIPTION
**Pull request by eroan-marie**

### CODE REVIEW

File: apps/jupyterhub/values.yaml [hunk 1]
```diff
@@ -0,0 +1,660 @@
...
+# ConfigurableHTTPProxy. KubeSpawner creates the user pods, and
+# ConfigurableHTTPProxy speaks with the actual ConfigurableHTTPProxy server in
+# the proxy pod.
+hub:
+  revisionHistoryLimit:
+  config:
+    GenericOAuthenticator:
+      client_id: jupyterhub
+      client_secret: {{ jupyterhub_oidc_client_secret }}
+      oauth_callback_url: https://staging1.{{ platform_domain_name }}/jupyter/hub/oauth_callback
```
Comment (nleconte-csgroup) [inline apps/jupyterhub/values.yaml]: Replace staging1 with processing or remove it.

File: apps/keycloak/keycloakrealmimport.yaml [hunk 2]
```diff
@@ -1077,6 +1107,127 @@ spec:
         policies: []
         scopes: []
         decisionStrategy: UNANIMOUS
+    - id: eb1767fa-57dc-462b-aa42-5bcdb543c323
+      clientId: jupyterhub
+      name: jupyterhub
+      description: ''
+      rootUrl: https://staging1.{{ platform_domain_name }}
```
Comment (nleconte-csgroup) [inline apps/keycloak/keycloakrealmimport.yaml]: Replace staging1 with processing

File: apps/keycloak/keycloakrealmimport.yaml [hunk 3]
```diff
@@ -1077,6 +1107,127 @@ spec:
...
+      adminUrl: https://staging1.{{ platform_domain_name }}
```
Comment (nleconte-csgroup) [inline apps/keycloak/keycloakrealmimport.yaml]: Replace staging1 with processing

File: apps/keycloak/keycloakrealmimport.yaml [hunk 4]
```diff
@@ -1077,6 +1107,127 @@ spec:
...
+      baseUrl: ''
+      surrogateAuthRequired: false
+      enabled: true
+      alwaysDisplayInConsole: false
+      clientAuthenticatorType: client-secret
+      secret: "{{ jupyterhub_oidc_client_secret }}"
+      redirectUris:
+      - ''
+      - https://staging1.{{ platform_domain_name }}/*
```
Comment (nleconte-csgroup) [inline apps/keycloak/keycloakrealmimport.yaml]: Replace staging1 with processing

File: apps/keycloak/keycloakrealmimport.yaml [hunk 5]
```diff
@@ -1077,6 +1107,127 @@ spec:
...
+      webOrigins:
+      - https://staging1.{{ platform_domain_name }}
```
Comment (nleconte-csgroup) [inline apps/keycloak/keycloakrealmimport.yaml]: Replace staging1 with processing

File: inventory/sample/group_vars/all/apps.yml [hunk 6]
```diff
@@ -53,3 +54,7 @@ prefect:
     password: "tmp_db_password_for_prefect"
     name: prefect
     secret: prefect-database-password
+
```
Comment (nleconte-csgroup) [inline inventory/sample/group_vars/all/apps.yml]: There is a merge conflict here. Rebase first.

**Review by nleconte-csgroup — CHANGES_REQUESTED** [review]
@eroan-marie some modifications are needed.

File: apps/jupyterhub/values.yaml (hunk 1 above)
Comment (eroan-marie) [reply apps/jupyterhub/values.yaml]: Done

File: inventory/sample/group_vars/all/apps.yml (hunk 6 above)
Comment (eroan-marie) [reply inventory/sample/group_vars/all/apps.yml]: Rebase done

**Review by nleconte-csgroup — APPROVED** [review]

**Pull request merged**